Hearthstone Deck Tracker/LogReader/Handlers/LoadingScreenHandler.cs

```diff
@@ -75,6 +75,13 @@ public void Handle(LogLine logLine, IHsGameState gameState, IGame game)
 				else
 					Watchers.DungeonRunWatcher.Stop();
 
+				if (game.CurrentMode == Mode.HUB || game.CurrentMode == Mode.TOURNAMENT || game.CurrentMode == Mode.ADVENTURE
```

Comment:
I feel like this should be moved into a prop (something like "MenuModes").

Reply:
Do you mean something like "PlayerCanBeChallenged"?

Reply:
Something like that, yes.